database/migrations/2019_07_12_045144_create_plans_table.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreatePlansTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        if (!Schema::connection('dbp_users')->hasTable('plans')) {
+            Schema::connection('dbp_users')->create('plans', function (Blueprint $table) {
+                $table->bigIncrements('id');
+                $table->string('name');
+                $table->boolean('featured')->default(false);
+                $table->integer('user_id')->unsigned();
+                $table->date('suggested_start_date');
+                $table->foreign('user_id', 'FK_plans')->references('id')->on(config('database.connections.dbp_users.database') . '.users')->onDelete('cascade')->onUpdate('cascade');
+                $table->softDeletes();
+                $table->timestamp('created_at')->default(DB::raw('CURRENT_TIMESTAMP'));
+                $table->timestamp('updated_at')->default(DB::raw('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'));
+            });
+        }
+
+        if (!Schema::connection('dbp_users')->hasTable('plan_days')) {
+            Schema::connection('dbp_users')->create('plan_days', function (Blueprint $table) {
+                $table->bigIncrements('id');
+                $table->bigInteger('plan_id')->unsigned();
+                $table->foreign('plan_id', 'FK_plan_days')->references('id')->on(config('database.connections.dbp_users.database') . '.plans')->onDelete('cascade')->onUpdate('cascade');
```

Comment:
ideally, we'd have some clear and consistent naming convention for foreign keys.  that does not seem to be the case currently.  eg FK_plan_days is just FK_(tablename) but links to plans (not evident in name).  FK_playlist_plan is better, as it mentions the elements being linked (playlists and plans).  i don't have perfect naming convention to suggest, and will approve this PR, but i did want to give this feedback that clear and consistent FK naming convention would be preferred.

Reply:
Jon changed to be more descriptive. Thanks